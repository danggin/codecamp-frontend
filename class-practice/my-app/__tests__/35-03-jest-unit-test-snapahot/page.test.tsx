import "@testing-library/jest-dom";
import JestUnitTestSnapshotPage from "@/app/section35/35-03-jest-unit-test-snapahot/page";
import { render } from "@testing-library/react";

it("기존 스냅샷이랑 바뀐게 없는지 비교 - 스냅샷 테스트", () => {
  const result = render(<JestUnitTestSnapshotPage />);

  expect(result.container).toMatchSnapshot();
});
