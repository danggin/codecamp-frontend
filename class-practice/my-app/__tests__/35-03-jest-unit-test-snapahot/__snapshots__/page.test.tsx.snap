// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 스냅샷이랑 바뀐게 없는지 비교 - 스냅샷 테스트 1`] = `
<div>
  <div>
    안녕하세요
  </div>
  취미 입력하기: 
  <input
    type="text"
  />
  <button>
    등록
  </button>
</div>
`;
